documentation/workflows.md

```diff
@@ -0,0 +1,74 @@
+# Working on the Project
+
+Better the be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
+
+## Principles
+
+1. No work is ever done directly on `main`; ALWAYS branch off of the target receiving branch (i.e. `main` for the majority of work)
+2. All changes must be reviewed and approved, and pass all quality gates
+
+## Development Workflows
+
+- [Starting Work](#starting-work)
+- [Pipelines](#pipelines)
+- [Changesets](#changesets)
+- [Approvals](#approvals)
+- [Local Testing](#local-testing)
+
+### Starting Work
+
+1. Pick a task from the issues/project
+    - Ask the team if anyone is working on the task
+    - Assign yourself to the taks
+2. Create a branch off of the target receiving branch (i.e. `main`)
+    - `feat/#-___` - for feature work
+    - `fix/#-___` - for hotfix work
+    - Relace "#" with the task number
+    - Replace "___" with a hyphenated breif title of the task
+3. Commit changes and push the branch
+    - If changes are being made to `src/*` files or "config" for artifact output a [Changesets](#changesets) is required.
+        - `pnpm changeset`
+        - Follow the prompts to create the changeset file for your changes.
+    - Follow the conventions of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for final (hopefully single) commit of the branch.
+    - Keep the branch current:
+        - Click the "Upudate Branch" button in the GitHub UI, *or*
+        - *(alternatively/optional) Use Rebase*
+            - `git checkout main`
+            - `git pull --rebase upstream main`
+            - `git checkout <branch>`
+            - `git rebase main`
+            - Resolve any conflicts that occur
+4. Open a Pull Request to the [central repository](github.com/gohypergiant/standard-toolkit) for review
+    - Add "close|closed|closes|fixed|fixes #12345" to the optional footers of a commit message; e.g. the example would close issue #12345.
```

Comment:
This will be part of the templates when they are implemented as well.

Reply:
It's also something that we could capture in the commit if we utilize something like `git-cz`

Reply:
I'll add a `.gitmessage` file to `.github/`. Configuration will still need to be done by each dev. https://gist.github.com/lisawolderiksen/a7b99d94c92c6671181611be1641c733#git-configuration

Reply:
This has been added to the template so isn't really needed anymore.

Reply:
The template works on issues and PRs in the GitHub UI - I assume - this is for commit messages made on the dev's machine... or am I missing something?